developer/src/kmcmpdll/kcframe/kcframe.vcxproj

```diff
@@ -114,7 +114,7 @@
     <ClCompile>
       <Optimization>MaxSpeed</Optimization>
       <InlineFunctionExpansion>OnlyExplicitInline</InlineFunctionExpansion>
-      <AdditionalIncludeDirectories>.;..\..\..\global\inc;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
These changes just so we can build the C++ host app 'kcframe' for kmcmpdll (used for debug currently) after we rearranged developer source files for 16.0.